bench/db_bench.ml

```diff
@@ -110,91 +124,88 @@ module Index = struct
 
   let read r () = Array.iter (fun (k, _) -> ignore (Index.find r k)) random
 
-  let write_random () =
+  let write_random name =
     Index_unix.reset_stats ();
-    let rw = Index.v ~fresh:true ~log_size (root // "fill_random") in
-    print_results (write rw) nb_entries;
```

Comment:
you can't reverse the order of `print_results` and `write_amplif` because `write_amplif` uses the stats gather during  the execution of the bench in `print_results`. I almost didn't see this ! Could you run the benchmarks before and after your changes to check that everything looks roughly the same?